docker-compose.test-custom.yml

```
@@ -61,6 +61,7 @@ services:
       IMAGE_STORAGE_SECRET_KEY: SECRET_KEY
       JSON_WEB_TOKEN_EXPIRY_MINUTES: 10080
       JSON_WEB_TOKEN_SECRET: purple
+      LOGS_HOST: logs_host.com
```

It'd be better to use
```suggestion
      LOGS_HOST: logs.balenadev.io
```
as it's a domain we control and less likely to cause accidental weirdness if we reach out to a random domain for testing (alternatively I think `*.example.com` is also safe?)

---

Changed to `logs.balenadev.io` in https://github.com/balena-io/open-balena-api/commit/6aa95b8696218eedaa5b1576948dafbd8887c9d9 👍🏼